website/content/community/_index.md

```diff
@@ -105,9 +105,14 @@ To develop MetalLB, you'll need a couple of pieces of software:
   tool)
 - [Docker](https://www.docker.com/docker-community), the container
   running system
+- [kind](https://github.com/kubernetes-sigs/kind), a lightweight Kubernetes cluster running in Docker
 - [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/), the Kubernetes commandline interface
 - [Invoke](https://www.pyinvoke.org) to drive the build system
 
+>NOTE: The development environment was tested with **kind `v0.7.0`**. Older
+>versions may not work since there have been breaking changes between minor
+>versions.
```

Comment:
mental note: hopefully now as the `kind` command we run are rather simple, they will work with newer releases too. So, for now, this note seems fine and no extra complications (like checks in the code, etc.) needed :-)